### PR TITLE
Add 4.5.0 to the table of releases

### PR DESCRIPTION
**Pull request by jovirkku**
Signed-off-by: Jorma Virkkunen <jorma.virkkunen@jolla.com>

### CODE REVIEW

**Review by jpwalden — APPROVED** [review]
Looks good to me

File: Support/Releases/README.md [hunk 1]
```diff
@@ -8,43 +8,44 @@ nav_order: 100
...
-| [4.3.0.15](https://forum.sailfishos.org/t/release-notes-suomenlinna-4-3-0/8495)      | Suomenlinna          | 10th Feb 2022 | **Stop release**            |
-| [4.2.0.21](https://forum.sailfishos.org/t/release-notes-verla-4-2-0/7092)            | Verla                | 16th Sep 2021 | **Stop release**            |
-| [4.1.0.24](https://forum.sailfishos.org/t/release-notes-kvarken-4-1-0/5942)          | Kvarken              | 27th May 2021 | **Stop release** &nbsp;Support for aarch64 |
-| [4.0.1.48](https://forum.sailfishos.org/t/release-notes-koli-4-0-1/4542)             | Koli                 | 16th Feb 2021 | **Stop release**            |
+| Release                                              | Name                 | Published     | Note                        |
+| -----------------------------------------------------| -------------------- | ------------- | --------------------------- |
+| [4.5.0.16](https://forum.sailfishos.org/t/14290/2)   | Struven ketju        | 2nd Feb 2023  | **Stop release:** deltarpm relies on zstd compression; AppSupport 11. |
```
Comment (vigejolla) [inline Support/Releases/README.md:13]: Drop the "/2" from forum link?

File: Support/Releases/README.md [hunk 2]
```diff
@@ -8,43 +8,44 @@ nav_order: 100
...
+| [4.4.0.72](https://forum.sailfishos.org/t/13110)     | Vanha Rauma          | 30th Sep 2022 | **Stop release:** enable zstd in deltarpm and related tools.<br />Browser engine esr78. Sailfish ported to Xperia 10 III. |
+| [4.3.0.15](https://forum.sailfishos.org/t/8495)      | Suomenlinna          | 10th Feb 2022 | **Stop release:** rpm update.<br />Signed Android system image.  |
+| [4.2.0.21](https://forum.sailfishos.org/t/7092)      | Verla                | 16th Sep 2021 | **Stop release:** preparing for signed Android AppSupport 10 on 4.3.0; adaptation repositories rectified. |
+| [4.1.0.24](https://forum.sailfishos.org/t/5942)      | Kvarken              | 27th May 2021 | **Stop release** Sailfish ported to Xperia 10 II. 64-bit architecture on Sailfish and on Android AppSupport. AppSupport 10 for XA2 & 10. |
```
Comment (vigejolla) [inline Support/Releases/README.md:17]: This line and the one below it have no ':' after "Stop release". Looks clumsy as the other lines do have the colons.

**Review by vigejolla — APPROVED** [review]

File: Support/Releases/README.md (hunk 2 above)
Comment (jovirkku) [reply Support/Releases/README.md:17]: The reason for the stop release is after the colon. Those two releases do not have the reason. Maybe not a clear way...

File: Support/Releases/README.md (hunk 1 above)
Comment (jovirkku) [reply Support/Releases/README.md]: Dropped

**Comment by jovirkku** [comment]
The bug used for tracking this task was actually JB#60065.

**Pull request merged**